cluster/src/main/scala/org/apache/spark/executor/SnappyExecutor.scala

```diff
@@ -108,6 +108,24 @@ class SnappyExecutor(
       }
     }
   }
+
+  override def isStoreCloseException(t : Throwable) : Boolean = {
+    try {
+      Misc.checkIfCacheClosing(t)
+    } catch {
+      case ex: CacheClosedException => true
```

Comment:
@sumwale great. This seems to be the reason for SNAP-1377. Because the ExecutorLostFailure exception got missed due to the check of only CacheClosedException.